samples/net/latmon/dts/bindings/latency-monitor.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Jorge Ramirez-Ortiz <jorge.ramirez@oss.qualcomm.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO latency monitor
+
+compatible: "latency-monitor"
+
+child-binding:
+  properties:
+    gpios:
+      type: phandle-array
+      required: true
+
+    pinctrl-0:
+      type: phandle
+      required: false
```

Comment:
This is not a device, these properties do not go in the devicetree bindings. If you need to specify gpios to use for your sample, use the special node `zephyr,user` and get the properties using `DT_PATH(zephyr_user)`